bundle/compliance/cis_azure/rules/cis_3_15/rule.rego

```diff
@@ -0,0 +1,16 @@
+package compliance.cis_azure.rules.cis_3_15
+
+import data.compliance.lib.common
+import data.compliance.policy.azure.data_adapter
+import data.compliance.policy.azure.storage_account.ensure_tls_version as audit
+
+finding = result {
+	# filter
+	data_adapter.is_storage_account
+
+	# set result
+	result := common.generate_result_without_expected(
+		common.calculate_result(audit.is_tls_configured("TLS1_2")),
```

Comment:
will it fail if the TLS version is higher than 1.2? I didn't see a test for this

Reply:
This is the highest available value.

Reply:
what if there will be newer versions in the future?

Reply:
Then this will be a different rule, because they explicitly check a certain version in the benchmark.

Reply:
```3.15 Ensure the "Minimum TLS version" for storage accounts is set to "Version 1.2" (Automated)```